fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -573,6 +573,7 @@ private void sendFragment() throws TException, RpcException, UserException {
                         LOG.warn("catch a execute exception", e);
                         exception = e;
                         code = TStatusCode.THRIFT_RPC_ERROR;
+                        BackendServiceProxy.getInstance().removeProxy(pair.first.brpcAddress);
```

Comment:
??

Reply:
We should remove proxy if rpc error happen